lalrpop/src/grammar/parse_tree.rs

```diff
@@ -902,7 +914,7 @@ impl Display for Symbol {
 impl Display for SymbolKind {
     fn fmt(&self, fmt: &mut Formatter) -> Result<(), Error> {
         match *self {
-            SymbolKind::Expr(ref expr) => write!(fmt, "{}", expr),
+                        SymbolKind::Expr(ref expr) => write!(fmt, "{}", expr),
```

Comment:
Nit: Some extra indentation here.

Reply:
All the indentation is to be fixed in upcoming work, that's why I call it work-in-progress